This patch changes nothing of how dumppfa.c works, but allows gcc-3.3.2 to build
without an ICE on ia64.

11 Jul 2006 agriffis

--- fontforge/dumppfa.c	2006-04-06 13:26:12.000000000 -0400
+++ fontforge/dumppfa.c.new	2006-07-11 19:14:40.000000000 -0400
@@ -118,7 +118,7 @@
 	    /* find some random bytes where at least one of them encrypts to */
 	    /*  a non hex character */
 	    r = 55665;
-	    cypher = ( randombytes[0] ^ (r>>8));
+	    cypher = ( randombytes[0] ^ (fed->r>>8));
 	    if ( isspace(cypher) )
 	goto try_again;
 	    if ( cypher<'0' || (cypher>'9' && cypher<'A') || (cypher>'F' && cypher<'a') || cypher>'f' )
